pkg/riff/commands/credential_set.go

```diff
@@ -17,25 +17,73 @@
 package commands
 
 import (
+	"bufio"
 	"context"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"syscall"
 
 	"github.com/projectriff/riff/pkg/cli"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
 	corev1 "k8s.io/api/core/v1"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type CredentialSetOptions struct {
 	cli.ResourceOptions
+	DockerHubId       string
+	DockerHubPassword string
+	GcrTokenPath      string
+	Registry          string
+	RegistryUser      string
+	RegistryPassword  string
+	SetAsDefault      bool
```

Comment:
Should the field name incorporate something to indicate it is about the default image prefix rather than some kind of default credential?

Reply:
Also, this isn't tested.

Reply:
Renamed to `--set-default-image-prefix`